modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -30,7 +30,7 @@ If the output URL scheme requires TLS (HTTPS, TLS, or UDPS), then TLS server-sid
 
 * `infrastructure`. Container logs from pods that run in the `openshift*`, `kube*`, or `default` projects and journal logs sourced from node file system.
 
-* `audit`. Logs generated by the node audit system (auditd) and the audit logs from the Kubernetes API server and the OpenShift API server. 
+* `audit`. Logs generated by the node audit system (auditd) and the audit logs from the Kubernetes API server and the OpenShift API server.
```

Comment:
the node audit system, called auditid, and the 

(We try to avoid parentheticals.)